docs/ProjectCommands.md

```diff
@@ -0,0 +1,78 @@
+# Project Command Reference
+
+Commands to do with projects. Remember that parameters with whitespace
+must be enclosed by quotation marks.
+
+## Options
+
+```sh
+/rocket project {list, view, help, create, unassign, edit, assign, delete}
+```
+
+### List
+
+```sh
+/rocket project list
+```
+
+Display a list of all projects.
+
+### View
+
+```sh
+/rocket project view PROJECT_ID
+```
+
+Displays details of project.
+
+### Help
+
+```sh
+/rocket project help
+```
+
+Displays options for `project` command.
+
+### Create (Team Lead and Admin only)
+
+```sh
+/rocket project create GH_REPO [--name DISPLAY_NAME]
+```
+
+Creates a new project from the given repo.
+
+### Unassign
+
+```sh
+/rocket project unassign PROJECT_ID
+```
+
+Unassigns the given project.
+
+### Edit
+
+```sh
+/rocket project edit PROJECT_ID [--name DISPLAY_NAME]
+```
+
+Edit the given project.
+
+### Assign
+
+```sh
+/rocket project assign PROJECT_ID GITHUB_TEAM_NAME [-f]
+```
+
+Assigns the project to the team. Fails if another team is assigned the project.
+If `-f` flag is given, can reassign even if
+another team is already assigned the project.
```

Comment:
Formatting: place on the line above (not really an issue for md, but just style in general)

Reply:
I split these because the style checker was complaining that my lines were too long.